_spinnaker_user_guides/deploying.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: Deploying (AWS EC2)
+order: 40
+redirect_from:
+  - /user-guides/deploying/
+  - /user_guides/deploying/
+  - /spinnaker_user_guides/deploying/
```

Comment:
Should this also include `/spinnaker-user-guides/deploying/`?

Reply:
Nope, this creates URL `/spinnaker-user-guides/deploying/` and doesn't need a self redirect.